notification-service/test/integration/notification_test.ts

```diff
@@ -0,0 +1,30 @@
+import request from 'supertest';
+import 'mocha';
+import {expect} from 'chai';
+import app from '../../src/app';
+import {Server} from "http";
+
+describe('Notification integration test', function () {
+    let server: Server;
+
+    before(function () {
+        server = app.listen(3000);
```

Comment:
Probably it would be better to use some ENV variable for PORT.

Reply:
We'll going to do it in the refactoring step of the TDD approach.